closed/autoconf/custom-hook.m4

```diff
@@ -340,6 +341,25 @@ AC_DEFUN([OPENJ9_PLATFORM_EXTRACT_VARS_FROM_CPU],
   esac
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_OJDKTHREAD_SUPPORT],
```

Comment:
Please move this addition after `OPENJ9_CONFIGURE_JITSERVER` so the definitions are (closer to being) sorted?

Reply:
sure.

Reply:
Re-ordered `AC_DEFUN`s.